pyriemann/transfer/pipelines.py

```diff
@@ -0,0 +1,949 @@
+import numpy as np
```

Comment:
this file does not really implement the pipeline object but more the estimators. I would therefore call it `_estimators.py` with _ as you are not supposed to import anything from there but I agree we are not consistent on this. It is just better practice